charts/nginx-gateway-fabric/values.yaml

```diff
@@ -105,12 +105,18 @@ terminationGracePeriodSeconds: 30
 ## Tolerations for the NGINX Gateway Fabric pod.
 tolerations: []
 
-##  The nodeSelector of the NGINX Gateway Fabric pod.
+## The nodeSelector of the NGINX Gateway Fabric pod.
 nodeSelector: {}
 
 ## The affinity of the NGINX Gateway Fabric pod.
 affinity: {}
 
+## The platform that NGINX Gateway Fabric is being deployed on. Valid values: kubernetes, openshift
+platform: kubernetes
+
+## Create and apply the OpenShift SecurityContextConstraints object.
+createOpenShiftSCC: true
```

Comment:
I wonder if in the future we will need to add any more openshift related things?
would it make sense to start a openshift field?

like

```yaml
openshift:
   createSCC: true
```

Reply:
I still don't think this field is necessary, per other comments?

Reply:
Would that require platform to be set to `openshift` or would we get rid of the `platform` field?

Reply:
we can detect if the platform is OpenShift without adding additional parameters, with something like
```
{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" }}
```

Reply:
Hm, yea per the above comments, it looks like I would just have the `platform` field. 

I'll make the change and remove the `createOpenShiftSCC` field for now and we can move forward with that.

Reply:
Wait, if we can detect if the platform is Openshift, do we need the `platform` field?

Reply:
Hm, that comment was not there when I commented, I'll look into that to see how it works and test it, if it works I'll move forward with that.

Reply:
Tested that the above works 👍  huge improvement.